Presentation.Web/Controllers/API/V1/OData/ItContractsController.cs

```diff
@@ -59,8 +60,9 @@ public IHttpActionResult GetItContracts(int key)
                 return Forbidden();
             }
 
-            var result = Repository.AsQueryable().ByOrganizationId(key);
-
+            var result = Repository.AsQueryable()
+                .Include(x => x.LastChangedByUser)
```

Comment:
no need to add include-- revert all changes in this file
Notice that none of the other properties must be pre-loaded in this code line..

It's the odata expand expressions that are translated into the right sql